src/code_b72b0_len_15ed0.c

```diff
@@ -9,17 +13,198 @@ extern s32 D_80154388; // entity fog alpha
 extern s32 D_8015438C; // entity fog dist min
 extern s32 D_80154390; // entity fog dist max
 
-INCLUDE_ASM(s32, "code_b72b0_len_15ed0", clear_entity_models);
+s32 step_entity_rendercmd(EntityModel* entityModel);
 
-INCLUDE_ASM(s32, "code_b72b0_len_15ed0", init_entity_models);
+void clear_entity_models(void) {
+    s32 i;
 
-INCLUDE_ASM(s32, "code_b72b0_len_15ed0", load_entity_model);
+    if (!gGameStatusPtr->isBattle) {
+        gCurrentEntityModelList = &gWorldEntityModelList;
+    } else {
+        gCurrentEntityModelList = &gBattleEntityModelList;
+    }
+
+    for (i = 0; i < MAX_ENTITY_MODELS; i++) {
+        (*gCurrentEntityModelList)[i] = NULL;
+    }
+
+    D_8015437C = 0xA;
```

Comment:
We've been using decimal for non-flag values, so I think all these numbres here and in init_entity_models should be changed